grafana: supply the explanation on the metrics of Flow-Control in TiKV. (#15720)
pingcap/docs: grafana-tikv-dashboard.md

![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)

### Flow Control

- Scheduler flow: The scheduler traffic on each TiKV instance in real time.

> - Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is not 0, it indicates that flow control exists. When `Compaction pending bytes` exceed its threshold, TiKV linearly increases the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry rejected requests automatically.

Reviewer: ```suggestion
- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio > 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```

> - Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is greater than 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler Discard Ratio` based on the exceeded portion. The client will retry the rejected requests automatically.

Reviewer 2: ```suggestion
- Scheduler discard ratio: The rejection ratio of scheduler requests on each TiKV instance. If this ratio is greater than 0, it indicates that flow control exists. When `Compaction pending bytes` exceeds its threshold, TiKV will linearly increase the `Scheduler discard ratio` based on the exceeded portion. The client will retry the rejected requests automatically.
```

Author: updated in https://github.com/pingcap/docs/pull/15765/files

> - Throttle duration: The block duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.

Reviewer: ```suggestion
- Throttle duration: The blocked duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
```

> - Throttle duration: The duration that the execution of the scheduler requests is blocked when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.

Author: ```suggestion
- Throttle duration: The block duration for the execution of the scheduler requests when flow control is triggered due to too many L0 files. If this metric has values, it indicates that flow control exists.
```

> - Scheduler throttled CF: The CF that triggers RocksDB throttling when the flow control threshold is reached
> - Flow controller actions: The actions that trigger RocksDB throttling when the flow control threshold is reached
> - Flush/L0 flow: The traffic of flush and L0 compaction for different CFs of RocksDB on each TiKV instance

Reviewer 3: ```suggestion
- Scheduler throttled CF: The CF that triggers RocksDB throttling when the flow control threshold is reached.
- Flow controller actions: The actions that trigger RocksDB throttling when the flow control threshold is reached.
- Flush/L0 flow: The traffic of flush and L0 compaction for different CFs of RocksDB on each TiKV instance.
```

- Flow control factors: The factors related to triggering RocksDB throttling.
- Compaction pending bytes: The size of the RocksDB data awaiting compaction in real time on each TiKV instance.

> - Txn command throttled duration: The block duration for commands related to transactions due to throttling. Under normal circumstances, this metric is 0.

Reviewer: ```suggestion
- Txn command throttled duration: The blocked duration for commands related to transactions due to throttling. Under normal circumstances, this metric is 0.
```

> - Non-txn command throttled duration: The block duration for other commands due to throttling. Under normal circumstances, this metric is 0.

Reviewer: ```suggestion
- Non-txn command throttled duration: The blocked duration for other commands due to throttling. Under normal circumstances, this metric is 0.
```

> - Non-txn command throttled duration: The blocked duration for other commands due to throttling. Under normal circumstances, this metric is 0.

Reviewer 2: 最后这两个，截图中没有

Author: updated it in https://github.com/pingcap/docs/pull/15765/files
